working/1426-extension-types/feature-specification-views.md

```diff
@@ -32,12 +32,12 @@ the set of members, with subsetting as a special case.
 
 The functionality is entirely static. Invocation of a view member is
 resolved at compile-time, based on the static type of the receiver.  Inside
-the view declaration, the scoping and the type and meaning of `this` is the
-same as for extension methods (a feature which was added to Dart in version
-2.6). This is important because it implies that the language Dart has a
-single and consistent semantics for all statically resolved member
-invocations, rather than having one set of rules for extension methods, and
-a different set of rules for view members.
+the view declaration, the reserved word `super` is treated as an identifier
+denoting a final variable whose type is the on-type of the enclosing view
+declaration. At run time, the value of `super` is the current on-type instance
+of the view. Similarly, the type of `this` is the view type declared by the
+enclosing declaration, and the run-time value of `this` is also the current
+on-type instance.
```

Comment:
This requires grammar changes.

Probably just making `super` a `primary`, where currently the primary contains:
```ebnf
  \alt \SUPER{} <unconditionalAssignableSelector>
  \alt \SUPER{} <argumentPart>
```
Then make it a compile-time error if `super` is not followed by an unconditinoalAssignableSelector or an argumentPart unless the `super` occurs inside a member declaration of a view.